### DISCUSSION TITLE
배경음악 적용법

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#921] [5. Q&A]
배경음악 bgm을 넣으려고 하는데 방법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 일반적인 유니티와 동일하게 Audio Source를 이용해서 추가할 수 있습니다.
무작정 따라하기 강의 Part6.에 관련 내용이 있으니 참고 부탁 드려요.
감사합니다. 

https://studio.zepeto.me/guides/world-tutorial-chapter1
